### PR TITLE
Replace exit events with close

### PR DESCRIPTION
**Pull request by ajay-gandhi**
Given the documentation for the [exit](https://nodejs.org/api/child_process.html#child_process_event_exit) event, it seems that the password may yet be outputted on `stderr` even after the `exit` event has been sent.

Therefore, replacing the exit event with [close](https://nodejs.org/api/child_process.html#child_process_event_close) will ensure that any data that would be sent to `stderr` is sent before the end.

In short, while using the `getPassword` function from this module I was receiving errors where the stderr `data` event was fired after the `exit` event on the child process, resulting in an error even though the password was indeed sent.

### CODE REVIEW

**Comment by mvhaen** [comment]
Sorry for the delay, I've been busy with some other stuff. Looks like a valid fix.

**Comment by ajay-gandhi** [comment]
Can you push the changes to NPM? Trying to use the package in one of my projects :p

**Comment by mvhaen** [comment]
Done.

**Comment by mvhaen** [comment]
version is now 0.0.6 on npm
